### PR TITLE
Remove legacy client

### PR DESCRIPTION
**Pull request by marians**
This PR removes the last parts of the old client.

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -251,7 +250,7 @@ func setParamsWithAuthorization(p *AuxiliaryParams, w *WrapperV2, params paramSe
 	setParams(p, w, params)
 }
 
-// CreateAuthToken creates an auth token using the latest client.
+// CreateAuthToken creates an auth token using the V2 client.
 func (w *WrapperV2) CreateAuthToken(email, password string, p *AuxiliaryParams) (*auth_tokens.CreateAuthTokenOK, error) {
 	params := auth_tokens.NewCreateAuthTokenParams().WithBody(&models.V4CreateAuthTokenRequest{
```
Comment (kopiczko) [inline client/client.go]: Unrelated but `auth_tokens`?

Comment (marians) [reply client/client.go]: What's the question? The sub-package `auth_tokens` is generated (part of gsclientgen) based on api-spec tags.

Comment (kopiczko) [reply client/client.go]: LoL

**Comment by marians** [comment]
@kopiczko Thanks for hanging in here! This was the last one. I based it off another branch, but didn't manage to get it separate when basing this PR on master later. The commits still appeared.

Anyway.

**Pull request merged**